src/main/java/com/btrapp/jklarfreader/KlarfReader.java

```diff
@@ -97,7 +97,7 @@ public static KlarfFormat findKlarfFormat(BufferedInputStream is) throws IOExcep
     // FileVersion 1 2
     is.mark(1000);
     final Pattern pattern18 = Pattern.compile("Record\\s+FileRecord.*1\\.8.*");
-    final Pattern pattern12 = Pattern.compile("FileVersion\\s+1\\s+2.*");
+    final Pattern pattern12 = Pattern.compile("FileVersion\\s+1\\s+[21].*"); // matches 1 1 or 1 2
```

Comment:
Should we rename this to pattern11or12 or something, or maybe just make 1.1 its own pattern, just for clarity?  If 1.1 and 1.2 work exactly the same pattern11or12 is probably better.